### ISSUE TITLE
Requirejs error when requiring.

### ISSUE THREAD

**Issue by ibebrett** [caolan/async#206]
It is currently discouraged to name the module with a define call. I was having issues when I installed the async library in say /static/js/libs/async, and had a config with a paths like this: 

paths: {
        libs: '/static/js/libs',
}

By removing the name from the define call at line 687 of async.js so that it looks like the below lline it worked perfectly. 

define([], function () {
            return async;
});

Basically what I am saying is that there are no benefits to naming it (at least I dont think there are), and it makes requiring it more fragile.
